src/server/db/connection.js

```diff
@@ -1,6 +1,12 @@
+const { types } = require('pg')
 const { POSTGRES_URL } = require('../environment')
 
 console.log('\nConnecting to database:', POSTGRES_URL)
+
+// override parser for date fields — just return the raw string content
+const DATE_OID = 1082
```

Comment:
where did the magic number come from?

Reply:
I followed Matt's research above to find this reference.

It looks like the constant is available via a separate package, `pg-types`.  I wasn't sure it was worth installing for a single constant, but we certainly can.  🤷